tests/testthat/tests_calculator_roc_class.R

```diff
@@ -0,0 +1,22 @@
+library(data.table)
+library(dimorfismo)
+library(testthat)
+
+calculator_roc <- roc$new()
+threshold_fake <- 0.285
+fake_data_to_test <- data.frame(
+  y_tests = c(1, 0, 1, 1, 0, 1, 0, 1, 0, 1),
+  prob = c(
+    9.287569, 3.667945, 9.398461, 1.513707, 8.624482,
+    9.996934, 1.176383, 9.548987, 2.338850, 6.769900
+  )
+)
+
+test_that("El método best_threshold_error funciona bien", {
+  expect_equal(calculator_roc$best_threshold_error(fake_data_to_test)$"threshold"[1], 0.505)
```

Comment:
Por favor encapsula en una variable el número mágico y el umbral obtenido. La prueba cambiaría a algo así:
```suggestion
  obtained_threshold <- calculator_roc$best_threshold_error(fake_data_to_test)$"threshold"[1]
  expected_threshold <- 0.505
  expect_equal(obtained_threshold, expected_threshold)
```